### PR TITLE
release: 2.4.1

### PR DESCRIPTION
**Pull request by pi0**
# Bugfixes 

* Handle `async` components correctly when using `nuxt-ts` (PR #4886)
* Fix `extractCSS` in `dev` mode (PR #4888)
* Correctly apply class when using `linkPrefetchedClass` (PR #4893)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.4.1](https://github.com/nuxt/nuxt.js/compare/v2.4.0...v2.4.1) (2019-01-29)
+
+
+### Bug Fixes
+
+* keepAliveProps broken in <nuxt-child> ([#4521](https://github.com/nuxt/nuxt.js/issues/4521)) ([b48b220](https://github.com/nuxt/nuxt.js/commit/b48b220))
+
```
Comment (manniL) [inline CHANGELOG.md]: Not sure whether we want to completely remove the top-level changelog or correctly update it

Comment (pi0) [reply CHANGELOG.md]: @manniL Lerna automatically does it because we forgot to update it.

Comment (manniL) [reply CHANGELOG.md]: Yeah, I know. But as we use `release notes` that could confuse ppl.

Comment (pi0) [reply CHANGELOG.md]: Reverted lerna version commit. We can gitignore top-level CHANGELOG.

**Review by TheAlexLichter — DISMISSED** [review]

**Comment by manniL** [comment]
Should we include https://github.com/nuxt/nuxt.js/pull/4892 ?

**Review by TheAlexLichter — CHANGES_REQUESTED** [review]
More fixes are available ☺️

**Review by TheAlexLichter — DISMISSED** [review]

**Pull request merged**